### PR TITLE
Negative correlation in event correlation (EQL) rules

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3497.

[Preview](https://security-docs_3709.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#create-eql-rule) - Made the following changes:
- **Step 1a**: Specified that the rule queries indices and data views for events (not alerts)
- **Step 1b**: Slightly revised the language and added a tip about using the syntax for negative correlation

### CODE REVIEW

**Review by paulewing — APPROVED** [review]
LGTM!

File: docs/detections/rules-ui-create.asciidoc [hunk 1]
```diff
@@ -206,8 +206,10 @@ IMPORTANT: Alerts created by threshold rules are synthetic alerts that do not re
 [[create-eql-rule]]
 ==== Create an event correlation rule
 . To create an event correlation rule using EQL, select *Event Correlation*, then:
-.. Define which {es} indices or data view the rule searches for alerts.
-.. Add an {ref}/eql-syntax.html[EQL statement] used to detect alerts.
+.. Define which {es} indices or data view the rule searches when querying for events.
+.. Write an {ref}/eql-syntax.html[EQL query] that searches for matching events or a series of matching events.
++
+TIP: When searching for a series of matching events, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax to specify events that should _not_ happen.
```
Comment (joepeeples) [inline docs/detections/rules-ui-create.asciidoc:212]: Maybe a different word at the end? It's unclear what it means if an event "should not happen." Does the syntax exclude events from the search?
```suggestion
TIP: When searching for a series of matching events, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax to specify events that should _not_ be matched.  
```

Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:212]: @joepeeples if I'm understanding the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/eql-syntax.html#eql-missing-events) correctly, negative correlation syntax should return events that don't match the conditions you specify. So it'd be like saying, "Find me all the events that _don't match_ this set of criteria." @paulewing @yctercero am I understanding this correctly? 

If I am, how does the following look:

```suggestion
TIP: To find events that don't match your specified conditions, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax.  
```

Comment (jmikell821) [reply docs/detections/rules-ui-create.asciidoc:212]: @nastasha-solomon I like this second option better. 👍

Comment (nastasha-solomon) [reply docs/detections/rules-ui-create.asciidoc:212]: Revising this based on @paulewing's feedback on Slack:

 ```suggestion
TIP: To find events that are missing in a sequence, use the {ref}/eql-syntax.html#eql-missing-events[missing events] syntax.  
```

**Review by yctercero — APPROVED** [review]

**Review by paulewing — APPROVED** [review]

**Pull request merged**